dotnet/src/Experimental/Assistants/Internal/Assistant.cs

```diff
@@ -111,13 +110,14 @@ public Task<IChatThread> GetThreadAsync(string id, CancellationToken cancellatio
     /// <param name="input">The user input</param>
     /// <param name="cancellationToken">A cancellation token.</param>
     /// <returns>An assistant response (<see cref="AssistantResponse"/></returns>
-    [KernelFunction, Description("Provide input to assistant a response")]
+    [KernelFunction, Description("Provide user message to assistant and retrieve the assistant response.")]
     public async Task<AssistantResponse> AskAsync(
-        [Description("The input for the assistant.")]
+        [Description("The user message provided to the assistant .")]
```

Comment:
nit: unnecessary space before the period.

Reply:
I'm all about whitespace management (among other things)